data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heals_affects_everybody.cfg

```diff
@@ -0,0 +1,87 @@
+#####
+# API(s) being tested: [heals]affect_allies=
+##
+# Actions:
+# Give the leaders a heals ability with affect_allies, affect_enemies, and affect_self.
+# Set all units' hitpoints to 1.
+# Wait a turn for healing to take place.
+##
+# Expected end state:
+# The Mage has 5 hp: 1 + 4 ([heals])
+#####
+{COMMON_KEEP_A_B_C_D_UNIT_TEST "heal_affect_everybody" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+                side = 1
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_HEALS 4 () ENEMIES=yes SELF=yes}
+                [/abilities]
+            [/effect]
+        [/modify_unit]
+
+        {SET_HP VALUE=1}
+
+        [end_turn][/end_turn]
+    [/event]
+
+    [event]
+        name = side 2 turn 1
+        [end_turn][/end_turn]
+    [/event]
+
+    [event]
+        name = side 3 turn 1
+        [end_turn][/end_turn]
+    [/event]
+
+    [event]
+        name = side 4 turn 1
+        [end_turn][/end_turn]
+    [/event]
+
+    [event]
+        name =side 1 turn 2
+        [end_turn][/end_turn]
+    [/event]
+
+    [event]
+        name = side 2 turn 2
+
+        # check self
+        [store_unit]
+            [filter]
+                side = 1
+            [/filter]
+            variable = temp1
+        [/store_unit]
+        {ASSERT {VARIABLE_CONDITIONAL temp1.hitpoints numerical_equals 7}}
+
+        # check ally
+        # not actually sure why this isn't 7 - rest healing should've applied and does appear to have happened when checking with --showgui
+        # but it's also not relevant for this test, so oh well
```

Comment:
Because rest healing happens during the turn refresh, and this event happens before the refresh.

Reply:
Updated.

Reply:
Just noticed that the second `s`, on the end of `affects`, does the same. The 3 other tests are `heal_affect_...`